src/hooks/useAccountStateBatch.tsx

```diff
@@ -11,7 +11,9 @@ export const isInAccountStateBatchCache = (
   addresses: GetAddressStateParams[]
 ) => {
   return addresses.every(({ chainId, address }) => {
-    return queryCache.find({ queryKey: ["accountState", chainId, address] });
+    return queryClientGlobal
```

Comment:
@henri-ly please check this part 🙏

Reply:
why do you need to do that ?
queryCache doesn't work ?

Reply:
it does, I just changed it